client/src/components/Tea/TeaList/TableFilter.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Datalist from "../../Datalist";
+
+export const TableFilter = props => {
+  return (
+    <div className="columns is-pulled-right">
+      <form onSubmit={() => props.handleFilterClick(event)}>
```

Comment:
You could simplify this to `onSubmit={props.handleFilterClick}` and avoid having to create a new anonymous function upon every render. Also, I think `event` is being pulled from the ether here.